pkg/deployments/tasks/20221101-l2-gauge-checkpointer/test/task.fork.ts

```diff
@@ -0,0 +1,234 @@
+import hre, { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { Contract } from 'ethers';
+
+import { BigNumber, fp } from '@balancer-labs/v2-helpers/src/numbers';
+import { GaugeType } from '@balancer-labs/balancer-js/src/types';
+import * as expectEvent from '@balancer-labs/v2-helpers/src/test/expectEvent';
+
+import { describeForkTest } from '../../../src/forkTests';
+import Task, { TaskMode } from '../../../src/task';
+import { getForkedNetwork } from '../../../src/test';
+import { impersonate } from '../../../src/signers';
+
+// This block number is before the manual weekly checkpoint. This ensures gauges will actually be checkpointed.
+describeForkTest('L2GaugeCheckpointer', 'mainnet', 15272610, function () {
```

Comment:
```suggestion
describeForkTest('L2GaugeCheckpointer', 'mainnet', 15835800, function () {
```

Let's do a more recent one

Reply:
Updated to a block right before last week's manual checkpoints.